aws/kops/Makefile

```diff
@@ -13,3 +13,39 @@ apply console destroy graph plan output providers show: init
 ## Pass arguments through to terraform which do not require remote state
 get fmt validate version:
 	terraform $@
+
+## Launch a shell with the kops environment
+kops/shell:
+	chamber exec kops -- bash -l
+
+## Render the kops manifest template
+kops/build-manifest:
+	build-kops-manifest
+
+## Apply the kops manifest
+kops/create:
+	kops create -f $(KOPS_MANIFEST)
+
+## Create the SSH Public Key secret
+kops/create-secret-sshpublickey:
+	kops create secret sshpublickey admin -i $(KOPS_SSH_PUBLIC_KEY_PATH)
```

Comment:
Where does `KOPS_SSH_PUBLIC_KEY_PATH` come from? Does there need to be some validation that this is set?

Reply:
Fair question. Documentation on this is [forthcoming](https://github.com/cloudposse/geodesic/pull/339/files#diff-6bfc873b2e29e553a63c1c61498f70ecR46) but lacks this env `KOPS_SSH_PUBLIC_KEY_PATH`.

Literal answer is here: https://github.com/cloudposse/terraform-root-modules/blob/kops-targets/aws/kops/main.tf#L99-L105